uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java

```diff
@@ -123,11 +127,34 @@ public static boolean isPackageInstalled(@NonNull final String packageName) {
         return false;
     }
 
+    /**
+     * Get a list of all brokers supported by our MSAL/ADAL sdks. These list contains all possible
+     * broker apps regardless of active build variant.
+     *
+     * @return a {@link List} of {@link ITestBroker} objects
+     */
     public static List<ITestBroker> getAllPossibleTestBrokers() {
         final List<ITestBroker> brokerList = new ArrayList<>();
         brokerList.add(new BrokerCompanyPortal());
         brokerList.add(new BrokerMicrosoftAuthenticator());
         brokerList.add(new BrokerHost());
         return brokerList;
```

Comment:
Nit:

return new Arrays.asList(new BrokerCompanyPortal(), new BrokerMicrosoftAuthenticator(), new BrokerHost());

Reply:
(it might need help to understand the type of the array, in which case new ITestBroker[]{...})

Reply:
Addressed